config/initializers/yabeda.rb

```diff
@@ -0,0 +1,37 @@
+# :nocov:
```

Comment:
This should not be needed here, because you added `add_filter('config')` to the simplecov config. 
Also, using `:nocov:` needs to wrap the code, so this would be missing a line at the end of the file: https://github.com/simplecov-ruby/simplecov?tab=readme-ov-file#ignoringskipping-code